packages/manager/apps/pci-instances/src/pages/instances/create/CreateInstance.page.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  PageLayout,
+  StepComponent,
+  Title,
+} from '@ovh-ux/manager-react-components';
+import { FC, useMemo } from 'react';
+import { useRouteLoaderData } from 'react-router-dom';
+import { TProject } from '@ovh-ux/manager-pci-common';
+import { useTranslation } from 'react-i18next';
+import { OsdsDivider } from '@ovhcloud/ods-components/react';
+import {
+  Breadcrumb,
+  TBreadcrumbProps,
+} from '@/components/breadcrumb/Breadcrumb.component';
+import { useHidePreloader } from '@/hooks/hidePreloader/useHidePreloader';
+
+const CreateInstance: FC = () => {
+  const project = useRouteLoaderData('root') as TProject;
+  const { t } = useTranslation(['create', 'common']);
```

Comment:
don't need to import if you do : common:create_instance

Reply:
To me it is better to import it, it is the way in the documentation